github-support/list-contributors.sh

```diff
@@ -20,7 +20,7 @@ CONTRIBUTORS_FILTERS="$DIR/FilteredNames.txt"
 
 cp $ALIASES $FILTERS $DIR && cd $DIR
 
-declare -a DEBEZIUM_REPOS=("debezium" "debezium-connector-db2" "debezium-connector-cassandra" "debezium-connector-vitess" "container-images")
+declare -a DEBEZIUM_REPOS=("debezium" "debezium-server" "debezium-operator" "debezium-connector-db2" "debezium-connector-cassandra" "debezium-connector-vitess" "debezium-connector-spanner" "debezium-connector-jdbc" "container-images")
```

Comment:
```suggestion
declare -a DEBEZIUM_REPOS=("debezium" "debezium-server" "debezium-operator" "debezium-connector-db2" "debezium-connector-cassandra" "debezium-connector-vitess" "debezium-connector-spanner" "debezium-connector-jdbc" "debezium-ui" "container-images")
```

Reply:
Added, I specifically left the `debezium-ui` repository out of the commit issue key check until things are officially integrated into our release pipeline like other sibling repositories to avoid any potential release issues.

Reply:
Ah, well then it makes sense to add it later after the integration.